MeNext/Playing/MainController.cs

```diff
@@ -251,7 +251,7 @@ public void RequestSkip()
             // TODO: songFinished with actual song
             var task = Task.Run(async () =>
              {
-                 return await api.SongFinished(this.EventSlug, this.UserKey, "dummy");
+                return await api.SongFinished(this.EventSlug, this.UserKey, this.musicService.PlayingSong.UniqueId);
```

Comment:
IDK, probably. ```SongFinished``` should only be called by the host and shouldn't have any restrictions on it. ```Skip``` should have some filtering in case several people skip a song.

We can make a new function, but we don't really *need* to right now...